terraform/alert_policy/missing_metric/variables.tf

```diff
@@ -0,0 +1,9 @@
+variable "alignment_period" {}
+variable "branch_suffix" {}
```

Comment:
should this be here? We can just add the suffix in the implementing repo and pass the full name right? See my comment above.